velox/connectors/hive/tests/HiveDataSinkTest.cpp

```diff
@@ -192,6 +205,58 @@ class HiveDataSinkTest : public exec::test::HiveConnectorTestBase {
     connectorQueryCtx_ = std::move(connectorQueryCtx);
   }
 
+  template <typename ReaderType, dwio::common::FileFormat format>
+  void testFlushPolicy() {
+    const auto outputDirectory = TempDirectoryPath::create();
+    auto flushPolicyFactory = []() {
+      if constexpr (format == dwio::common::FileFormat::PARQUET) {
+#ifdef VELOX_ENABLE_PARQUET
```

Comment:
All these ifdefs make the code hard to follow. Is there a cleaner way? CC: @majetideepak

Reply:
@mbasmanova Updated.